BloomGameStudio/BloomDiscordBot
# Updated contributor emojis & contributor listing

**Whats new**
- Modified /contributors message to display as two messages instead of one.
- Modified second message to not display contributors as individual line items

**How to test**
- Load up bot in test server
- invoke /contributors command
- Observe result

**Previous display**
![image](https://github.com/user-attachments/assets/54a60e49-8ba2-4f08-81ce-0c20ed0b3bb1)

**Updated Display (example)**
![image](https://github.com/user-attachments/assets/60e7e1a5-1924-442a-8f2d-0a20eb56197a)
